plugins/source/notion/docs/tables/notion_databases_table.md

```diff
@@ -0,0 +1,28 @@
+# Table: notion_databases_table
+
+This table shows data for Notion Databases Table.
+
+The primary key for this table is **_cq_id**.
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_id (PK)|`uuid`|
+|_cq_parent_id|`uuid`|
+|object|`utf8`|
+|id|`utf8`|
```

Comment:
shouldn't ID be the PK here?

Reply:
yes it should. I will be changing that in next commit.